lib/rialto/etl/logging.rb

```diff
@@ -12,9 +12,11 @@ module Logging
       def logger
         @logger ||= Yell::Logger.new(:null).tap do |logger|
```

Comment:
Given you're already touching a lot of our logging in this branch, I wonder if we can simplify this to:

```ruby
  @logger ||- Yell.new do |logger|
    # ...
  end
```

That is the incantation as described in the Yell README. Does invoking it with `:null` and tapping into it get us something? (Maybe a @justinlittman question.)